cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -1265,40 +1299,44 @@ pub(crate) mod tests {
             [install]
             vim.pkg-path = "vim"
             emacs.pkg-path = "emacs"
-            emacs.systems = ["system2"]
+            emacs.systems = ["aarch64-darwin" ]
 
             [options]
-            systems = ["system1",]
+            systems = ["x86_64-linux"]
         "#};
         let manifest = toml::from_str(manifest_str).unwrap();
 
-        let expected_params = vec![
-            PackageGroup {
-                name: DEFAULT_GROUP_NAME.to_string(),
-                system: "system1".to_string(),
-                descriptors: vec![PackageDescriptor {
+        let expected_params = vec![PackageGroup {
+            name: DEFAULT_GROUP_NAME.to_string(),
+            descriptors: vec![
+                PackageDescriptor {
                     allow_pre_releases: None,
-                    attr_path: "vim".to_string(),
+                    attr_path: "emacs".to_string(),
                     derivation: None,
-                    install_id: "vim".to_string(),
+                    install_id: "emacs".to_string(),
                     version: None,
-                }],
-            },
-            PackageGroup {
-                name: DEFAULT_GROUP_NAME.to_string(),
-                system: "system2".to_string(),
-                descriptors: vec![PackageDescriptor {
+                    allow_broken: None,
+                    allow_unfree: None,
+                    allowed_licenses: None,
+                    systems: vec![SystemEnum::Aarch64Darwin],
+                },
+                PackageDescriptor {
                     allow_pre_releases: None,
-                    attr_path: "emacs".to_string(),
+                    attr_path: "vim".to_string(),
                     derivation: None,
-                    install_id: "emacs".to_string(),
+                    install_id: "vim".to_string(),
                     version: None,
-                }],
-            },
-        ];
+                    allow_broken: None,
+                    allow_unfree: None,
+                    allowed_licenses: None,
+                    systems: vec![SystemEnum::X8664Linux],
```

Comment:
suggestion(nb): since we are practiacally only using one system here, in the current ungrouping impl, we could/should change this to `system`.
However, `PackageDescriptor` is a typealias to a generated type so we can't change its structure without adding a shim type or changing the schema upstream